docs/UserGuide.md

```diff
@@ -154,6 +155,24 @@ Examples:
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
+---
+### Editing a show: `Edit`
+
+Edit the specified show from Trackermon.
+
+Format: `edit <INDEX> [n/<NAME>] [s/<STATUS>] [t/<TAG>]…​`
+* Edit the show at the specified `<INDEX>`.
+* The index refers to the index number shown in the displayed show list. (not overall)
+* The index **must be a positive integer** 1,2,3,..
+* At least one field to edit must be provided
+* Editing to an existing name is not allowed
+
+Examples:
+* `list` followed by `edit 2 n/Sailor Moo` edit 2nd show's name in Trackermon to Sailor Moon.
```

Comment:
Should this be `edit 2 n/Sailor Moon` instead?

Reply:
Sailor Moo is a show about magical cow saving the world

Reply:
Time to change the Sailor Moon in `Trackermon to Sailor Moon` to Sailor Moo then haha.